### PR TITLE
Refactoring, cleaning up code, adding comments

### PR DESCRIPTION
**Pull request by Undercoverdog**
Hai there~! (*＾▽＾)／

I've made some super kawaii changes to our code~! uwu 💖✨

Here's a quick rundown:

- Replaced the boring ol' `if-elif` with a super cool `switch-case`! (´｡• ᵕ •｡`) ♡
- Reversed some changes to keep things in line~ (っ´▽`)っ
- Fixed the indentation to make everything neat and tidy! (๑˃̵ᴗ˂̵)و
- Did some cleaning up~ (*^‿^*)
- Added several comments for better understanding and did some more cleaning up! (๑>ᴗ<๑)
- Replaced that cringy `if-else` with something much better~! (*≧ω≦)
- Added comments, try-catch blocks, and other cool stuff! \(≧▽≦)/

I hope senpai @SpamixOfficial  notices my hard work! (⁄ ⁄•⁄ω⁄•⁄ ⁄)⁄

Looking forward to your feedback! Let's make our code sparkle together~! ✧◝(⁰▿⁰)◜✧

put the pull request in nice and deep

Arigatou gozaimasu~! (´｡• ω •｡`) ♡

### CODE REVIEW

File: linux/cli.py [hunk 1]
```diff
@@ -455,7 +465,7 @@
 ## Start of clicker code
 shouldClick = False # controlls the constantclick
 
-
+## No idea why you get a bug here hehe not my code not my problem @spamix
```
Comment (SpamixOfficial) [inline linux/cli.py]: What Bug?

**Review by SpamixOfficial — CHANGES_REQUESTED** [review]
Please explain what bug you're referring to?

File: linux/cli.py (hunk 1 above)
Comment (Undercoverdog) [reply linux/cli.py]: I dont really know, i just know that i got an error message when running it with Python

Comment (Undercoverdog) [reply linux/cli.py]: ...And i didnt change anything there so its your fault :)

**Comment by Undercoverdog** [comment]
@SpamixOfficial have you tested your code yourself last time before pushing?

**Comment by SpamixOfficial** [comment]
> @SpamixOfficial have you tested your code yourself last time before pushing?

Yes?

File: linux/cli.py (hunk 1 above)
Comment (Undercoverdog) [reply linux/cli.py]: @SpamixOfficial I get error `Key.f2 to click constantly (toggle on/off by clicking the key!)
Esc to exit!
Traceback (most recent call last):
  File "/home/user/Documents/coole sachen/openclick/linux/cli.py", line 469, in <module>
    @listener.event
     ^^^^^^^^
NameError: name 'listener' is not defined. Did you mean: 'Listener'?`

when running `python linux/cli.py `

Comment (SpamixOfficial) [reply linux/cli.py]: I'll take a look later when I get some time

Comment (Undercoverdog) [reply linux/cli.py]: I just checked your last push, everthing works there.
I dont see anything where I changed anything related to that event, thats odd.

Comment (SpamixOfficial) [reply linux/cli.py]: Weird

**Comment by SpamixOfficial** [comment]
@Undercoverdog I won't approve this pull request until you resolve the bug.

**Comment by Undercoverdog** [comment]
@SpamixOfficial yes i will fix it dw

**Review by SpamixOfficial — CHANGES_REQUESTED** [review]
Fix bug you were talking  about, since you managed to introduce it. Btw sorry for sounding aggressive, for some reason all my texts sounds aggressive :(

File: linux/cli.py (hunk 1 above)
Comment (Undercoverdog) [reply linux/cli.py]: @SpamixOfficial I just checked again, your last push on dev is causing the error. So its not my fault hehe.

`python cli.py`

```python


  /$$$$$$                                 /$$$$$$  /$$ /$$           /$$
 /$$__  $$                               /$$__  $$| $$|__/          | $$
| $$  \ $$  /$$$$$$   /$$$$$$  /$$$$$$$ | $$  \__/| $$ /$$  /$$$$$$$| $$   /$$
| $$  | $$ /$$__  $$ /$$__  $$| $$__  $$| $$      | $$| $$ /$$_____/| $$  /$$/
| $$  | $$| $$  \ $$| $$$$$$$$| $$  \ $$| $$      | $$| $$| $$      | $$$$$$/
| $$  | $$| $$  | $$| $$_____/| $$  | $$| $$    $$| $$| $$| $$      | $$_  $$
|  $$$$$$/| $$$$$$$/|  $$$$$$$| $$  | $$|  $$$$$$/| $$| $$|  $$$$$$$| $$ \  $$
 \______/ | $$____/  \_______/|__/  |__/ \______/ |__/|__/ \_______/|__/  \__/
          | $$
          | $$
          |__/


							SpamixOfficial 2023
Controls: 
Key.f1 to click (hold to click!) 
Key.f2 to click constantly (toggle on/off by clicking the key!)
Esc to exit!
Traceback (most recent call last):
  File "/tmp/OpenClick/linux/cli.py", line 460, in <module>
    @listener.event
     ^^^^^^^^
NameError: name 'listener' is not defined. Did you mean: 'Listener'?


```

Comment (SpamixOfficial) [reply linux/cli.py]: Well, would you mind changing it to "Listener" then?

**Comment by Undercoverdog** [comment]
@SpamixOfficial i commented the event out since they were always causing trouble. (As mentioned here) <https://github.com/SpamixOfficial/OpenClick/pull/25#discussion_r1536778360>

 Everything seems to work now. You can merge pull request now!

**Comment by SpamixOfficial** [comment]
Have you also tested so everything actually works correctly?

**Comment by Undercoverdog** [comment]
@SpamixOfficial i tested both f1 and f2 modes and they work. color works too.

**Pull request merged**